src/components/biggive-tipping-slider/biggive-tipping-slider.tsx

```diff
@@ -0,0 +1,117 @@
+import { Component, Element, Prop, h } from '@stencil/core';
+import { spacingOption } from '../../globals/spacing-option';
+import { brandColour } from '../../globals/brand-colour';
+
+@Component({
+  tag: 'biggive-tipping-slider',
+  styleUrl: 'biggive-tipping-slider.scss',
+  shadow: true,
+})
+export class BiggiveTippingSlider {
+  @Element() host: HTMLBiggiveTippingSliderElement;
+
+  @Prop() spaceBelow: spacingOption = 0;
+
+  @Prop() colourScheme: brandColour = 'primary';
+
+  @Prop() percentageStart: number;
+
+  @Prop() percentageEnd: number;
+
+  @Prop() donationAmount: number;
+
+  @Prop() donationCurrency: string = '£';
```

Comment:
I think we shouldn't have a default set for `donationCurrency`. If we have a default set then it's easy to omit passing it, and in future if/when we support donations in other currencies we could easily forget to pass them - can we do this:

```suggestion
  @Prop() donationCurrency: string;
```

(I wish we could get the compiler to force us to always pass a currency, which this won't do with current settings, but a missing currency symbol is still better than a wrong currency symbol.

Or maybe this to show that we currently only support once currency but there's room for expansion. If/when we support more currencies we can add them to the type and refactor to share type with anything else that needs it:

```suggestion
  @Prop() donationCurrency: '£';
```

Reply:
Updated as per suggestion